README.md

```diff
@@ -75,6 +76,12 @@ The [coreJSON](https://github.com/FreeRTOS/coreJSON) library is a JSON parser th
 
 See memory requirements for the latest release [here](https://docs.aws.amazon.com/embedded-csdk/202011.00/lib-ref/libraries/standard/coreJSON/docs/doxygen/output/html/index.html#json_memory_requirements).
 
+#### corePKCS11
+
+The [corePKCS11](https://github.com/FreeRTOS/corePKCS11) library is a software based implementation of the PKCS #11 interface (API) to enable rapid development and flexibility when developing applications that rely on cryptographic operations. Only a subset of [PKCS #11 v2.4](http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html) has been implemented, with a focus on operations involving asymmetric keys, random number generation, and hashing.
```

Comment:
The [corePKCS11] library is an implementation of the PKCS #11 interface (API) that makes it easier to develop applications that rely on cryptographic operations.